src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form  class="form" 
+      action="https://www.w3schools.com/action_page.php" 
+      method="post"
+    >
+      <fieldset class="container" >
+        <legend>Personal information</legend>
+        <label class="form-fild">
+          Surname:
+          <input type="text" 
+            name="Surname" 
+            autocomplete="off"
+          >
+        </label>
+        <label class="form-fild">
+            Name:
+          <input type="text" 
+            name="Name" 
+            autocomplete="off" 
+          >
+        </label>
+        <label for="age" 
+          class="form-fild"
+        >
+            How old are You?
```

Comment:
```suggestion
        <label for="age" class="form-fild">
          How old are You?
```
correct formatting :arrow_up: